modules/Documentation/components/HowItWorks/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Text from '../../../../components/Text';
+import Button from '../../../../components/Button';
+import { Container, Code } from '../../index.styled';
+import Snippet from '../Snippet';
+
+export default ( {ILP_SERVER_URL, clientId} ) => {
```

Comment:
Same like above

Also consider adding `eslint` to your IDE for automatic code formatting